_includes/manuals/3.16/1.2_release_notes.md

```diff
@@ -4,16 +4,222 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+#### User Interface & Experience
+- **All Hosts Page Redesign** - Continued modernization of the All Hosts page with enhanced functionality, including changing owner, location/organization, disassociating VMs, and other improvements.
+- **Job Invocations Detail Page** - Complete overhaul of job execution details with improved performance, better loading, and enhanced user interface
+
+#### Infrastructure & Platform Updates
+- **NodeJS 22 Upgrade** - Updated UI build stack to NodeJS 22, extending lifecycle support to April 2027
+- **PatternFly 5 Migration** - Modernization of UI components by upgrading from PatternFly 3 to PatternFly 5 in Foreman core
+- **New Installer 1.0 Release** - Major release of updated installation tooling
+
+#### Networking & Connectivity
+- **Dual-Stack IPv4/IPv6 Registration Support** - Full support for client registration in dual-stack environments, enabling seamless registration over both IPv4 and IPv6 networks
+
+#### Data & Analytics
+- **Insights Vulnerability Service Integration** - Support for Technology Preview version of Red Hat Insights Vulnerability service
+- **Advisor Recommendations Display** - Integration of Red Hat Insights Advisor recommendations and details within Foreman interface
+
+#### Documentation & Tooling
+- **Provisioning Documentation Improvements** - Enhanced documentation for provisioning workflows and best practices
+
 ### Upgrade warnings
 
+#### Recommended Pre-Upgrade Actions
+- **Audit RHV/oVirt Usage** - If using RHV compute resources, plan migration to alternative platforms before upgrading
+
 ### Deprecations
 
+- **🚨 RHV/oVirt Support Completely Removed** - Red Hat Virtualization (RHV) and oVirt compute resource support has been fully removed
+  - **Previous Deprecation**: Feature was deprecated in Foreman 3.3
+  - **Impact**: Users with RHV/oVirt compute resources will lose this functionality
+  - **Action Required**: Must migrate to alternative virtualization platforms before upgrading
```

Comment:
In 3.15 THEY SHOULD run `foreman-rake ovirt:drop`

Now in 3.16, it's automatic and part of the installation. Rake description:

```
    Remove all oVirt / RHV compute resources & profiles from Foreman.
    Hosts associated with these compute resources will be disassociated and kept.
    Host groups with oVirt compute resources & profiles will have their compute resource and compute profile removed.
    Compute profiles, images, and other resources will be removed.
```